docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/deployment.md

```diff
@@ -241,72 +253,155 @@ Follow these instructions to update devices.
+
+        
+        a. Enter your chosen credentials name and then execute the command to set the mapping name.
+
+        !!! note
+            To edit credentials, this command may be skipped, and then use one of the existing credential mappings in the following commands.
+
+        ```bash
+        curl --data '<creds-name>' -H "X-Consul-Token:<consul-token>" -X PUT "http://localhost:8500/v1/kv/edgex/v3/device-onvif-camera/Writable/InsecureSecrets/credentials001/SecretName"
+        ```
+        
+        b. Enter your chosen credentials name and then execute the command to create the mapping.
+        ```bash
+        curl -H "X-Consul-Token:<consul-token>" -X PUT "http://localhost:8500/v1/kv/edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/<creds-name>"
+        ```
+
+        c. Enter your chosen username, password, and authentication mode and credentials name and then execute the command to create the mapping.
+        !!! note
+            The options for authentication method are: `usernametoken`, `digest`, or `both`
+
+        ```bash
+        curl --data '{
+                    "apiVersion":"v3",
+                    "secretName": "<creds-name>",
+                    "secretData":[
+                        {
+                            "key":"username",
+                            "value":"<username>"
+                        },
+                        {
+                            "key":"password",
+                            "value":"<password>"
+                        },
+                        {
+                            "key":"mode",
+                            "value":"<auth-mode>"
+                        }
+                    ]
+                }' --header 'Authorization:Bearer <JWT-token>' -X POST "http://localhost:59984/api/v3/secret"
+        ```
+
+        d. Enter your mac-address(es) and then execute the command to add the mac address(es) to the mapping.
+        !!! note
+            If you want to map multiple mac addresses, enter a comma separated list in the command
+
+        ```bash
+        curl --data '<mac-address>' -H "X-Consul-Token:<consul-token>" -X PUT "http://localhost:8500/v1/kv/edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/<creds-name>"
+        ```
+        
+        e. Check the status of the credentials map.
+        ```bash
+        curl -H "X-Consul-Token:<consul-token>" -X GET "http://localhost:8500/v1/kv/edgex/v3/device-onvif-camera/AppCustom/CredentialsMap?keys=true" | jq .
+        ```
+        Example response:
+        ```bash
+        [
+        "edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/NoAuth",
+        "edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/credentials001",
+        "edgex/v3/device-onvif-camera/AppCustom/CredentialsMap/credentials002"
+        ]
+        ```
+
+        f. To query the mappings, enter the credentials name in this command and execute it.
```

Comment:
I say we can just rename it. I think it might be useful to for people to reference what macs are attached to what creds, and for verification purposes.

Reply:
yeah agreed, since the previous command did not do what we wanted

Reply:
here is a command we could give them to query all of the keys and values. not sure if it would be too confusing for the user:

```
for x in $(curl --no-progress-meter -H "X-Consul-Token:<consul-token>" -X GET "http://localhost:8500/v1/kv/edgex/v3/device-onvif-camera/AppCustom/CredentialsMap?keys=true" | jq '.[]' | tr -d '"'); do \
    printf "%20s" "$(basename $x): "; \
    curl -H "X-Consul-Token:<consul-token>" --no-progress-meter -X GET "http://localhost:8500/v1/kv/$x?raw=true"; \
    echo; \
done
```

Reply:
That's a tough one. I think that is probably too much.